message_ix/macro.py

```diff
@@ -215,8 +259,14 @@ def read_data(self):
             if name == "config":
                 continue
             idx = _validate_data(
-                name, self.data[name], self.nodes, self.sectors, self.years
+                name, self.data[name], self.nodes, self.sectors, self.levels, self.years
             )
+            self.units[name] = self.data[name]["unit"].mode().any()
```

Comment:
@behnam-zakeri here `any()` returns a boolean value (`True` or `False`), that is later inserted into some data frames, per the previous comment. Can you please clarify what the intent of this line is?

Reply:
I am guessing it is intended to get a single unique unit value.

Reply:
Yes, it's intended to get the unit that is used most, and use it as a single value.